test/helpers/contracts.js

```diff
@@ -7,6 +7,11 @@ function getLogs(Event, filter, additionalFilters) {
   return promisify(query.get.bind(query))();
 }
 
+const parseContractName = contractName => {
```

Comment:
What's the benefit of this vs split(v) ? I'm no good with regex, so does this expression satisfy if we have say `contractV10_12_999-beta`?

Reply:
Good catch.